darwin/torch/transforms.py

```diff
@@ -191,7 +198,6 @@ def __call__(self, image: PILImage.Image, target: TargetType) -> Tuple[PILImage.
         boxes = [obj["bbox"] for obj in annotations]
         # guard against no boxes via resizing
         boxes = torch.as_tensor(boxes, dtype=torch.float32).reshape(-1, 4)
-        boxes[:, 2:] += boxes[:, :2]
         boxes[:, 0::2].clamp_(min=0, max=w)
         boxes[:, 1::2].clamp_(min=0, max=h)
```

Comment:
This is not clamping bboxes anymore, since now indexes 2 and 3 are height and width instead of x2 and y2. Boxes now could go outside of the image.